api-reference/10 UI Components/dxOverlay/1 Configuration/copyRootClassesToWrapper.md

```diff
@@ -5,7 +5,28 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Copies your custom CSS classes from the root element to the wrapper element. 
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+This property allows you to use CSS for wrapper element configuration. You can customize the shade, resize, and drag zones.
+
+
+The following markup illustrates relations between wrapper and root elements when you set **copyRootClassesToWrapper** to **true**:
+
+    <!--HTML--><body>
+        
+        <!-- The following is the root element on which the component is initialized. -->
```

Comment:
- `The following` The following what?
- `on which the component is initialized` How can something be initialized **on** something else?

Reply:
>The following what?

We have comments on line 18 and 19 that describe what's going on in the code on line 20. 'The following' refers to the `<div ... ></div>` element.

>How can something be initialized on something else
It is a common thing. We have an element in markup (`<div`) and a component is 
'attached' to it when the component initialized. But when the component is used in the application, it is automatically 'attached' to some other element (usually `<body>`).

You can also find similar constructions in other docs (for example, Adobe):
![image](https://user-images.githubusercontent.com/33040934/127837231-b7e68fc6-d056-47e5-ba07-8312ede8dc2f.png)
![image](https://user-images.githubusercontent.com/33040934/127837252-c3d4191a-ff24-4e1e-9f42-a9c0f06a6537.png)

Reply:
1) The 'following' is incomplete, whereas the 'following element' is complete
2) Yes, I've seen this before in Adobe and some other documentation, however, it doesn't mean it is correct. I've seen countless questionable phrases and wording in Microsoft docs too. But ok, if you feel comfortable with this wording, then use it.